eslint.config.mjs

```diff
@@ -524,7 +524,7 @@ export default [
   },
   {
     name: 'mocha/recommended',
```

Comment:
Every time we upgrade an ESLint plugin from a version that didn't have a `name` property, we need to check if the new one has one. In this case it did, so this line could have been removed. I'm going to remove it in a fixup PR.